src/panels/config/integrations/integration-panels/zha/dialog-zha-restore-backup.ts

```diff
@@ -0,0 +1,368 @@
+import {
+  css,
+  CSSResultGroup,
+  html,
+  LitElement,
+  TemplateResult,
+  nothing,
+} from "lit";
+import { customElement, property, state } from "lit/decorators";
+import { mdiFileUpload } from "@mdi/js";
+
+import { fireEvent } from "../../../../../common/dom/fire_event";
+
+import { extractApiErrorMessage } from "../../../../../data/hassio/common";
+import {
+  ZHANetworkBackup,
+  createZHANetworkBackup,
+  listZHANetworkBackups,
+  fetchZHANetworkSettings,
+  restoreZHANetworkBackup,
+  ZHANetworkSettings,
+} from "../../../../../data/zha";
+
+import {
+  showAlertDialog,
+  showConfirmationDialog,
+} from "../../../../../dialogs/generic/show-dialog-box";
+
+import { createCloseHeading } from "../../../../../components/ha-dialog";
+import { haStyle, haStyleDialog } from "../../../../../resources/styles";
+import { HomeAssistant } from "../../../../../types";
+
+import { HaRadio } from "../../../../../components/ha-radio";
+import { HaCheckbox } from "../../../../../components/ha-checkbox";
```

Comment:
```suggestion
import type { HaRadio } from "../../../../../components/ha-radio";
import type { HaCheckbox } from "../../../../../components/ha-checkbox";
import from "../../../../../components/ha-radio";
import from "../../../../../components/ha-checkbox";
```

This makes sure to also import the element itself